tests/resources/data/baseline_output_ftrs_10columns.csv

```diff
@@ -0,0 +1,3987 @@
+file: data/ftrs_test_data.ftrs masses_file: data/masses/e_coli_monomer_masses.csv modifications: - Sodium - Potassium - Anhydro - DeAc - Deacetyl_Anhydro - Nude - Decay - Amidation - Amidase - Double_Anh - multimers_Glyco ppm: 10 rt_window: 0.5 ,ID,xicStart,xicEnd,ionCount,chargeOrder,rt,mwMonoisotopic,theo_mwMonoisotopic,inferredStructure,maxIntensity
```

Comment:
How / where is this file used?

Reply:
Good spot, thanks. Its not used, now removed.